doc/src/reference.xml

```diff
@@ -686,8 +688,8 @@ path-constant DATA : data/a.txt ;
         <listitem>
           <para><emphasis role="bold">Allowed values:</emphasis> <literal>on</literal>, <literal>off</literal>.</para>
 
-          <para>The <literal>runtime-debugging</literal> feature specifies if
-          produced object files, executables and libraries should include
+          <para>The <literal>runtime-debugging</literal> feature specifies that,
+          if produced, object files, executables, and libraries should include
```

Comment:
I see now that you are correct and that 'specified whether produced object files...' etc. is the correct terminology. I will make the change.